docs/changelog/133493.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 133493
+summary: Gate disable norms on text fields for logsdb/tsdb indices
+area: Mapping
+type: bug
```

Comment:
Not sure this needs to be an entry in the changelog, since it's fixing a bug in as-of-yet unreleased code. Maybe change the label from `>bug` to `>non-issue`, and remove this changelog entry?